IO/Export/vtkOBJExporter.cxx

```diff
@@ -124,17 +125,17 @@ void vtkOBJExporter::WriteAnActor(vtkActor *anActor, FILE *fpObj, FILE *fpMtl,
 {
   vtkDataSet *ds;
   vtkPolyData *pd;
-  vtkGeometryFilter *gf = NULL;
+  vtkNew<vtkGeometryFilter> gf;
```

Comment:
For performance and for a minimally invasive fix, I would not instantiate the filter if not needed.
So 2 choice: 
  - use a smart pointer instead of vtkNew
  - or declare+instantiate vtkNew only where needed (inside if)